<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qemu</class>
 <widget class="QMainWindow" name="qemu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEMU</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>QMP</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,1">
       <item>
        <widget class="QLineEdit" name="le_qmp_ip">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_qmp_port">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_qmp_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_logo">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_simstate">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Status</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_simstate">
             <property name="text">
              <string>State:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_time">
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="out_simstate">
             <property name="text">
              <string>&lt;font color=&quot;grey&quot;&gt;N/A&lt;font&gt;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="out_time">
             <property name="text">
              <string>&lt;font color=&quot;grey&quot;&gt;N/A&lt;font&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_version">
             <property name="text">
              <string>Version:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="out_version">
             <property name="text">
              <string>&lt;font color=&quot;grey&quot;&gt;N/A&lt;font&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_file_settings"/>
    <addaction name="action_file_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_help_about"/>
   </widget>
   <widget class="QMenu" name="menu_run">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="action_run_play"/>
    <addaction name="action_run_stop"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_tools_memdump"/>
    <addaction name="action_tools_memtree"/>
    <addaction name="action_tools_cpuregs"/>
    <addaction name="action_tools_asm"/>
    <addaction name="action_tools_logging"/>
    <addaction name="action_tools_timing"/>
   </widget>
   <widget class="QMenu" name="menu_plugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_run"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_plugins"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_help_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action_run_play">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="action_run_stop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_file_settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_tools_memdump">
   <property name="text">
    <string>Memory Dump</string>
   </property>
  </action>
  <action name="action_tools_memtree">
   <property name="text">
    <string>Memory Tree</string>
   </property>
  </action>
  <action name="action_tools_cpuregs">
   <property name="text">
    <string>CPU Registers</string>
   </property>
  </action>
  <action name="action_tools_asm">
   <property name="text">
    <string>Assembly</string>
   </property>
  </action>
  <action name="action_tools_logging">
   <property name="text">
    <string>Logging</string>
   </property>
  </action>
  <action name="action_tools_timing">
   <property name="text">
    <string>Timing</string>
   </property>
  </action>
  <action name="action_plugin_qmp">
   <property name="text">
    <string>QMP Debug</string>
   </property>
  </action>
  <action name="actionPlugins_2">
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
